ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -121,6 +121,7 @@ - (RTCVideoTrack *)createVideoTrack:(NSDictionary *)constraints {
     NSArray *videoDevices = [AVCaptureDevice devicesWithMediaType:AVMediaTypeVideo];
     for (AVCaptureDevice *device in videoDevices) {
         [devices addObject:@{
+                             @"facing": device.position,
```

Comment:
AFAIS Position is an NSInteger, we need a "front" or "back" string instead: https://developer.apple.com/documentation/avfoundation/avcapturedeviceposition?language=objc

Reply:
Thanks you.
It's correct now? I can't try in a real phone actually, so please try it if you can.